genon/preprocessor/facade/intelligent_processor.py

```
@@ -2901,65 +3105,97 @@ def get_level_name(level_num: int) -> str:
+            build_docling_document,
+            build_tabular_vectors,
+        )
+        if self._xlsx_cfg["processing_mode"] == "tabular":
+            _log.info(f"[intelligent] xlsx tabular 직접 처리: {file_path}")
+            vectors = build_tabular_vectors(
+                file_path,
+                header_row=self._xlsx_cfg["header_row"],
+                multi_table=self._xlsx_cfg["multi_table"],
+            )
+            if not vectors:
+                raise GenosServiceException(1, f"chunk length is 0")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)